akka-actor/src/main/scala/akka/actor/dungeon/Dispatch.scala

```diff
@@ -35,8 +35,11 @@ final case class SerializationCheckFailedException private[dungeon] (msg: Object
 @InternalApi
 private[akka] trait Dispatch { this: ActorCell =>
 
+  // scalafmt breaks this by moving the _ to its own line which doesn't compile
```

Comment:
Interaction of breaking long lines in scalafmt with the semicolon inference in at least Scala 3.3.1